tomster-player/tests/acceptance/list-albums-test.js

```diff
@@ -1,46 +1,27 @@
 import { test } from 'qunit';
 import moduleForAcceptance from 'tomster-player/tests/helpers/module-for-acceptance';
-import Pretender from 'pretender';
 import { authenticateSession } from 'tomster-player/tests/helpers/ember-simple-auth';
 
-const ALBUMS = [{
-  id: '1',
-  type: 'album',
-  attributes: {
-    title: 'The Bodyguard',
-    'cover-url': 'https://i.scdn.co/image/7bcb439989b592287aeac1599aaa36be17672b73'
-  }
-},
-{
-  id: '2',
-  type: 'album',
-  attributes: {
-    title: 'Whitney Houston',
-    'cover-url': 'https://i.scdn.co/image/3c61cf6b053cea492f3962b0b005b4e170afa37a'
-  }
-}];
+const { FIXTURES, loadFixtures, unloadFixtures } = window;
```

Comment:
Also the fixtures should probably be handled as ES6 modules as well:

```es6
import { albums } from '../fixtures/albums';
```

or so